packages/widget-v2/src/state/swapExecutionPage.ts

```diff
@@ -38,9 +40,10 @@ export const chainAddressesAtom = atom<Record<number, ChainAddress>>({});
 export const swapExecutionStateAtom = atomWithStorage<SwapExecutionState>(
   "swapExecutionState",
   {
-    route: undefined,
+    route: {} as RouteResponse,
```

Comment:
I think it's ok to make it undefined, any reason to do this?

Reply:
yeah i forget why i did this tbh, I think having it be undefined makes sense